### DISCUSSION TITLE
The Barsuk, the hardest "beginner" sub

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#8033] [Unstable feedback]
So the Barsuk is actually the hardest starter sub. 

-No fabricator/deconstructor so it can't craft
-No medical fabricator
-~~Only one operational gun out of 3~~ The back gun is now installed by default making the sub better
-Only one supercapacitor
-It's not really fast

You're left with a sub that can't fend for itself against the most basic enemies due to terrible gun coverage.

If this is supposed to be a starter sub that you switch for another later in the campaign, **you waste money equipping it with guns.** The Dugong and Humpback are better equipped from the get go and you don't need to spent 5000+ mks to catch up.
![image](https://user-images.githubusercontent.com/73229309/149420786-c0f2cdf5-9958-4e70-bfd9-0f46aa372600.png)

Instead I suggest pre-equipping the Barsuk with two starter coilguns with nerfed stats: half the fire rate and worse handling...
![image](https://user-images.githubusercontent.com/73229309/149422361-de4e4910-f9db-4a7a-b010-7baeff401692.png)

This makes it a true beginner sub, no bad economics, just a disposable small starter sub that pushes you to replace it.

[Barsuk but not as bad.zip](https://github.com/Regalis11/Barotrauma/files/7866499/Barsuk.but.not.as.bad.zip)

**Comment by Ek-Videogames** [comment]
The Barsuk makes the **v9.0 Beta Humpback look polished**.

**Comment by rav2n** [comment]
In my opinion, adding such a small sub is the right decision. Simply because it is more convenient for 1-2 players.
In addition, all scout submarines do not have fabricators. This is fine.

**Reply by LePiromano** [reply]
orca, orca 2, and azimuth have fabricators and are scout class

**Reply by rav2n** [reply]
> orca, orca 2, and azimuth have fabricators and are scout class

Oops. Although I know why I wrote that)

**Comment by LePiromano** [comment]
if the dugong and the humpback were married barsuk would be the result

honestly i think it's fine and i agree with (the first part) of what rav4n said

**Comment by Udrakan** [comment]
It's a fun starter sub for me personally. I like maneuvering around the blindspots to hit critters with laser. After 2 or 3 levels it might become tiresome, but at that time you save enough cash for a regular sub with crafting facilities, which is helped by having more credits at the start of campaign with this cheap sub.

I am concerned, however, about it being a noob trap. It's cheap, layout feels easy to understand from the splash pic in the menu and that might make someone completely new start their early experience with Barotrauma on it, frustrated that they can't deal with the enemies in the blindspots. I would totally add something in the description about difficulty of starter subs.

**Comment by ubertpendragon** [comment]
Note that the back gun has been added by default since the last unstable, so the sub is much less punishing.
